src/main/scala/diplomaticobjectmodel/model/OMISA.scala

```diff
@@ -37,6 +43,7 @@ case class OMISA(
   u: Option[OMSpecification],
   s: Option[OMSpecification],
   addressTranslationModes: Seq[OMAddressTranslationMode],
+  customExtensions: List[OMCustomExtensionSpecification],
```

Comment:
I'd make this a `Seq`, since you don't actually care about the underlying `scala.collection` data structure implementation

Reply:
Changed

Reply:
changed